dataprofiler/profilers/utils.py

```diff
@@ -483,3 +483,18 @@ def find_diff_of_dicts(dict1, dict2):
         diff = "unchanged"
 
     return diff
+
+def get_capacity(data, unit='M'):
+    """
+    Get size (capacity) of the input data
+
+    :param data: list or array of data
+    :type data: Union[list, numpy.array, pandas.DataFrame]
+    :return: capacity of the input data
+    """
+    unit_map = {'B': 0, 'K': 1, 'M': 2, 'G': 3}
```

Comment:
nice, do we want to library specific error if someone provides a value which is not in the dict?

Reply:
added